components/asana/sources/new-workspace/new-workspace.mjs

```diff
@@ -17,15 +18,17 @@ module.exports = {
     },
```

Comment:
I think you can also create a webhook making use of [workspace resource type](https://developers.asana.com/docs/workspace) instead of polling workspaces

Reply:
Yes, I tried, but don't worked, this is the reason for listing all workspaces to emit the events.

I thing this is outdated in the API docs.